frontend/public/js/control.js

```diff
@@ -158,36 +159,91 @@ function sendMove(){
 	console.log(pan + " - " + tilt);
 }
 
+/* Variables used for the zoom slider */
+var zoomSend = false;
+var zoom = 0;
+
 /**
-* Method to send the new input value of the zoom slider to the currently selected camera.
+* Method is called when the inputslider value changes.
 */
-function inputzoomslider(zoom) {
-	$.get("/api/backend/camera/" + currentcamera + "/zoom?zoomType=absolute&zoom=" + zoom , function(data) {});
-	console.log("Zoom: " + zoom);
+function inputzoomslider(z) {
+	if (zoomSend === false) {
+		zoom = z;
+		zoomSend = true;
+		setTimeout(function(){sendZoom(); zoomSend = false;}, 130);
+		sendZoom();
+	} else {
+		zoom = (parseInt(z) + parseInt(zoom)) / 2;
+	}
 }
 
 /**
+* Method to send a command to the backend to change the zoom.
+*/
+function sendZoom() {
+	$.get("/api/backend/camera/" + currentcamera + "/zoom?zoomType=absolute&zoom=" + parseInt(zoom) , function(data) {});
+	console.log("Zoom: " + parseInt(zoom));
+}
+
+/* Variables used for the focus slider */
+var focus = 0;
+var focusSend = false;
+
+/**
 * Method to send the new input value of the focus slider to the currently selected camera.
 * It also change the status of the auto focus.
 * @param focus value of the new input.
 */
-function inputfocusslider(focus) {
+function inputfocusslider(f) {
 	$('#auto_focus').addClass("btn-danger");
 	$('#auto_focus').removeClass("btn-success");
-	$.get("/api/backend/camera/" + currentcamera + "/focus?autoFocusOn=false&position=" + focus , function(data) {});
-	console.log("Focus: " + focus);
+	if (focusSend === false) {
+		focus = f;
+		focusSend = true;
+		setTimeout(function(){sendFocus(); focusSend = false;}, 130);
+		sendFocus();
+	} else {
+		focus = (parseInt(f) + parseInt(focus)) / 2;
```

Comment:
@Kingdorian Yes, I agree this is duplicate code. I changed it, do you agree with merging now?